src/Microsoft.Diagnostics.Runtime/src/Linux/Structs/ElfFileTableHeader32.cs

```diff
@@ -8,9 +8,9 @@
 namespace Microsoft.Diagnostics.Runtime.Linux
 {
     [StructLayout(LayoutKind.Sequential)]
-    public struct ElfFileTableHeader
+    public struct ElfFileTableHeader32
```

Comment:
Should this be public? I think `internal` is more appropriate unless you need it for something?

Reply:
I agree it should be internal.